src/security/BlindPeerConnector.cc

```diff
@@ -76,7 +76,7 @@ Security::BlindPeerConnector::noteNegotiationDone(ErrorState *error)
         // based on TCP results, SSL results, or both. And the code is probably not
         // consistent in this aspect across tunnelling and forwarding modules.
```

Comment:
FWIW, we are testing changes that address the above XXX. They are independent from this fix.